### ISSUE TITLE
Speed&&Accuracy

### ISSUE THREAD

**Issue by www516717402** [SakuraRiven/EAST#8]
Hello，thanks for your shre！
I meet two questions need your help.
When I trained AdvanceEAST by Keras, it process quickly and result is accuracy. However. I trained slowly by your project and result is not accuracy.
About 3500 images with input train. double 1080Ti spend 24 hours. But AdvanceEAST spend 1 hours.

**Comment by www516717402** [comment]
Hello，thanks for your share！
your project is perfect，But still have some questions.
1. Input None label data abort.
2. detect stage，RGBA trans RGB
3. train very slowly.
4. result low accuracy.
Modify data process and loss that solve these question.
Finally. The Loss about 5 epoch，detect accuracy 80%. I also modify model to 15MB.

**Comment by www516717402** [comment]
Have already solved by [CSDN BLOG](https://blog.csdn.net/u011046017/article/details/93392862)

**Comment by SakuraRiven** [comment]
Hi, This repo is for ICDAR2015, so there are some details which could be changed according to your own dataset. Here is an example:

The training batch is cropped from image, which would change the distance GT. I was lazy to handle such case, so I just repeat 1000 times to crop image to avoid split text region. The text regions in ICDAR2015 are often sparse and this takes little time. However, the time consumption would increase when dense texts exist. This may be the reason of low speed. ( I guess...

**Comment by lionelcinnamon** [comment]
"The training batch is cropped from image, which would change the distance GT" 

I have a question:
In your code, you had cropped image before getting ground truth distance, so it is not affect the result of ground truth distance, right?
